app/src/main/java/org/simple/clinic/summary/teleconsultation/sync/TeleconsultationSync.kt

```diff
@@ -0,0 +1,53 @@
+package org.simple.clinic.summary.teleconsultation.sync
+
+import com.f2prateek.rx.preferences2.Preference
+import io.reactivex.Completable
+import io.reactivex.Single
+import org.simple.clinic.main.TypedPreference
+import org.simple.clinic.main.TypedPreference.Type.LastTeleconsultationFacilityPullToken
+import org.simple.clinic.sync.ModelSync
+import org.simple.clinic.sync.SyncConfig
+import org.simple.clinic.sync.SyncCoordinator
+import org.simple.clinic.user.UserSession
+import org.simple.clinic.util.Optional
+import javax.inject.Inject
+import javax.inject.Named
+
+class TeleconsultationSync @Inject constructor(
```

Comment:
We need to add `TeleconsultationSync` class to the `SyncModule#syncs` provider method for the sync to get triggered.